languages/en.ini

```diff
@@ -109,8 +109,13 @@ Availability = "Availability"
 Available = "Available"
 Available Functionality = "Available Functionality"
 Awards = "Awards"
+Back to Author Browse = "Back to Author Browse"
```

Comment:
I wonder if it would be better to use a tokenized string, like `back_to_browse = "Back to Browse %%browse%%"` instead of creating a separate language string for every browse value. Then you could inject the existing translated browse handler names in as the token, instead of having to duplicate everything. If this sounds like a good idea but you're not sure what I'm talking about, let me know and I can provide more detail. :-)

Reply:
It's true that some adjustments to the way things are labeled may be necessary in some languages to make a tokenized approach work well... but I think it's a solvable problem, and probably easier to fix that than to maintain a larger number of strings. :-)

Reply:
I predict some translation issues, so a question: Does the back link really need the browse index, or could it be just "Back to Browse" or "Back to Results"?

Reply:
@EreMaijala, do you think it would help if we just changed `Back to Browse %%browse%%` to `Back to Browse (%%browse%%)`? That way, we're not building a sentence like "Back to Browse By Title" but are simply providing additional context like "Back to Browse (By Title)". I think that would probably work with a larger number of possible constructions, and could be easily customized if the original form was preferred and workable. But I also don't object to removing the fuller context if @rominail doesn't think it makes a difference.

I'd also be interested in your opinion on the "separate back-link vs. breadcrumb trail" thread elsewhere on this PR.

Reply:
That'd be easier to translate, but I'm still not sure it's worth all the complexity.

I don't think this can be integrated with breadcrumbs since they can be disabled, but perhaps this should only be displayed when breadcrumbs are disabled?

Reply:
See my [comment from last week](https://github.com/vufind-org/vufind/pull/3826#discussion_r1696930666): I think it can be integrated with breadcrumbs on the record page, which I think is the only place that matters. But maybe I'm forgetting/overlooking something!

Reply:
I'm having trouble thinking of the scenario where breadcrumbs would be disabled on the record page, unless the administrator really doesn't want breadcrumbs there. Since this essentially behaves like a breadcrumb, I'm not sure if we would want it to override that preference. But I do agree that the text would need to be reworded to eliminate "Back to" if we decided to move it into that context.

Reply:
We currently have breadcrumbs disabled and display a backlink on the record page instead: 
![kuva](https://github.com/user-attachments/assets/7508627a-87e2-4d95-86d3-2a5a7265e9a4)

So if you wanted something like that, integrating with breadcrumbs wouldn't quite work. I also understood that @rominail wanted this to be separate from breadcrumbs. But if breadcrumbs were enabled, I'd try to make sure not to display duplicate links either.